### DISCUSSION TITLE
How to burn liquidswap lp tokens using their smart contract??

### DISCUSSION THREAD

**Discussion by ajaythxkur** [aptos-labs/aptos-developer-discussions#527] [Questions] [answered]
After adding tokens to pool, i want to completely burn the lp tokens received (doesn't mean remove liquidity)

**Comment by gregnazario** [comment, marked as answer]
### How to burn a Coin in three ways
#### 1. I have access to the `BurnCapability`
* I can burn them with `0x1::coin::burn` if I have access to this.  Likely you don't unless you created the LP token.

#### 2. I'm a programmer and I want to lock it in a vault and throw away the key
* I can write a simple immutable contract that works as a vault

```move
module addr::vault {
  use aptos_framework::coin;
  struct Vault<MyCoin> {
      coins: Coin<MyCoin>
  }

  entry fun lock_forever<MyCoin>(caller: &signer, amount: u64) {
     let coins = coin::withdraw<MyCoin>(caller, amount);
     move_to(caller, Vault<MyCoin>{
         coins
     });
  }
}
```

#### 3. Transfer them to an address that no one can access

This is most used in the community for things that don't apply to number 1

Just transfer to a burn address (usually used on Aptos as `0xFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF`).

**Reply by ajaythxkur** [reply]
Thanks @gregnazario
